lte/protos/subscriberdb.proto

```diff
@@ -168,7 +168,6 @@ message CoreNetworkType {
       NT_EPC = 0;
       NT_5GC = 1;
   }
-  // Network Types to be restricted per subscriber. If not configured, Subscriber will have access to all Network Types by default.
```

Comment:
@ulaskozat The proto file is not latest, will rebase it and amend the changes. Thanks for your comment.

Reply:
Rebased it